src/main/java/org/jitsi/videobridge/cc/vp8/VP8FrameMap.java

```diff
@@ -75,9 +76,47 @@ private FrameInsertionResult doFrameInsert(@NotNull VP8Frame frame, Vp8Packet pa
             logger.warn(e);
         }
         frame.addPacket(packet);
-        return new FrameInsertionResult(frame, false);
+        return new FrameInsertionResult(frame, false, false);
     }
 
+    /** Check whether this is a large jump from previous state; if so, reset the map. */
```

Comment:
The way this is phrased makes me expect the method to do the reset, but it doesn't.